frontend/app/models/attendance.js

```diff
@@ -20,11 +20,14 @@ export default Model.extend({
 
 , activityDuration: computed('activities.@each.from', 'activities.@each.to', {
     get() {
+      return 0
+      /*
       return moment.duration(this.get('activities').reduce((duration, activity) => {
```

Comment:
Here, we should probably query the store for the activities in the same date range as the attendance, should be pretty straightforward through a computed property or even directly accessing the store.